test/e2e/operator/deployment_api.go

```diff
@@ -46,9 +46,9 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// We use intentionally use this non-existing driver image
+// We use intentionally use this non-existing service point for driver image
```

Comment:
"service point" - "container registry" probably is a better name.

Reply:
changed this and deleted duplicated "use"